src/lib/createElement__v2.js

```diff
@@ -7,5 +11,86 @@ export function createElement__v2(vNode) {
 //    - 요소 생성
 //    - 속성 설정 (이벤트 함수를 이벤트 위임 방식으로 등록할 수 있도록 개선)
 //    - 자식 요소 추가
-  return {}
+
+
+// 1. falsy vNode 처리
+if (!vNode) {
+  return document.createTextNode('');
+}
```

Comment:
```suggestion
if (vNode === undefined || vNode === null || typeof vNode === 'boolean') {
  return document.createTextNode('');
}
```
오늘 솔루션 코드 시간에 코치님께서 말씀해주신 부분인데, 이렇게 처리하면 숫자 0이 빈 문자열로 렌더링되는 문제가 있다고 하시더라고요.

저도 !vNode로 처리하고 있었는데, 이 부분은 수정이 필요할 것 같아요 😂